src/status_im/translations/ne.cljs

```diff
@@ -0,0 +1,314 @@
+
+   ;;phone types
+   :phone-e164                            "अन्तराष्ट्रिय १"
+   :phone-international                   "अन्तराष्ट्रिय २"
+   :phone-national                        "राष्ट्रिय"
+   :phone-significant                     "महत्वपूर्ण"
+
+   ;;chats
+   :chats                                 "च्याटहरु"
+   :new-chat                              "नया च्याट"
+   :delete-chat                           "च्याट मेटाउने"
+   :new-group-chat                        "नया सामूहिक च्याट"
+   :new-public-group-chat                 "सार्वजनिक च्याटमा भाग लिने"
+   :edit-chats                            "च्याटहरुको सम्पादन"
+   :search-chats                          "च्याटहरुमा खोज"
+   :empty-topic                           "खाली विषय"
+   :topic-format                          "गलत ढाँचा [a-z0-9\\-]+"
+   :public-group-topic                    "विषय"
+
+   ;;discover
+   :discover                              "भेट्टाउनुहोस्"
+   :none                                  "बिना कुनै"
+   :search-tags                           "खोज्ने ट्यागहरु यहाँ टाइप गर्नुहोस्"
+   :popular-tags                          "लोकप्रिय ट्यागहरु"
+   :recent                                "भर्खरको"
+   :no-statuses-discovered                "कुनैपनि स्टेटस भेट्टाइएन"
+   :no-statuses-found                     "कुनैपनि स्टेटस फेलापारिएन "
+
+   ;;settings
+   :settings                              "सेटिङ्गहरु"
+
+   ;;contacts
+   :contacts                              "सम्पर्कहरु"
+   :new-contact                           "नया सम्पर्क"
+   :delete-contact                        "सम्पर्क मेटाउने"
+   :delete-contact-confirmation           "यो सम्पर्क तपाईंको सम्पर्कहरुबाट हटाइने छ"
+   :remove-from-group                     "समूह बाट हटाउने"
+   :edit-contacts                         "सम्पर्क सम्पादन"
+   :search-contacts                       "सम्पर्कहरुमा खोज"
+   :show-all                              "सबै देखाउने"
+   :contacts-group-dapps                  "डीएयापहरु"
+   :contacts-group-people                 "मानिसहरु"
+   :contacts-group-new-chat               "नया च्याट सुरु"
+   :choose-from-contacts                  "सम्पर्कहरुबाट चयन गर्नुहोस्"
+   :no-contacts                           "हाल कुनै सम्पर्कहरु छैनन्"
+   :show-qr                               "QR देखाउने"
+   :enter-address                         "ठेगान प्रविष्ट गर्ने"
+   :more                                  "अरु"
+
+   ;;group-settings
+   :remove                                "हटाउनुहोस्"
+   :save                                  "साँच्नुहोस्"
+   :delete                                "मेटाउनुहोस्"
+   :change-color                          "रंग बदल्नुहोस्"
+   :clear-history                         "इतिहास खाली गर्नुहोस्"
+   :mute-notifications                    "सूचनाहरु चुप बनाउनुहोस्"
```

Comment:
I think मौन relates closely to meaning of "mute"

in android, the options we get when pressing and holding notifications uses these

[] सूचनाहरूलाई बिना आवाज देखाउने
[] सबै सूचनाहरूलाई रोक्नुहोस्
[] मौन नगर्नुहोस् वा नरोक्नुहोस्

so there's a simple one रोक्नुहोस् too. 

regarding "real physical person" aren't we already treating computers as anthropomorphised beings? we are as polite to them as we are to real people already.